lib/otel/rules.test.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2024 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+// Based upon https://github.com/open-telemetry/opentelemetry-js/blob/8fc76896595aac912bf9e15d4f19c167317844c8/packages/opentelemetry-sdk-trace-base/test/common/Span.test.ts#L851
+
+const test = require('node:test')
```

Comment:
is this test even getting run? it's not in the tests folder

Reply:
It is not getting run automatically at this time. I only wanted to be able to verify the work while we hash out the structure and underlying details. My intention is to relocate it as we draw near to it being a real feature.

Reply:
got it